minio/console
# Fix Set Policy on Groups with spaces

It was not possible to set a policy on a group with spaces:

<img width="816" alt="Screenshot 2024-10-14 at 1 48 04 PM" src="https://github.com/user-attachments/assets/55be84b8-512a-4b6a-9e22-b83d861c31f8">

We had a regex to validate the group name in the request but current UI doesn't allow the user to define that manually, the groups should already exists, so technically is not necessary. Right now this regex is wrong cause it doesn't allow spaces and it should. We could fix it but it but it might not allow other stuff, here we can rely on the server to return that error if something was wrong.

<img width="684" alt="Screenshot 2024-10-14 at 2 53 29 PM" src="https://github.com/user-attachments/assets/94d7a14a-f9dd-4a7b-804c-d9dac0dcb52d">


### Test Steps:

1. First:
- Create a group with spaces like e.g. `some group`
- Click on new group then Policies
- Click on Assign policies
- Assign policies and click save
- All should work


3. Also: Creating two groups
- In Listing Policies
- Select the two new groups
- Select ASsign policies
- Select policies to attach 
- Save (should work fine)
